src/geo/projection/globe_util.js

```diff
@@ -189,6 +189,11 @@ export function globeTileBounds(id: CanonicalTileID): Aabb {
     return Aabb.fromPoints(corners);
 }
 
+export function interpolateVec3 (from: Vec3, to: Vec3, phase: number): Vec3 {
```

Comment:
Nit: style, not sure why eslint isn't picking this up:

```suggestion
export function interpolateVec3(from: Vec3, to: Vec3, phase: number): Vec3 {
```